### PR TITLE
fix: fix for the hearing booking bug

### PR DESCRIPTION
**Pull request by toireasam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2249

### Change description ###
Changing e2e test data to reference new test data as old was causing issue with creating hearing bookings as old test data did not contain sdo directions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/fixtures/caseData/gatekeepingNoHearingDetails.json [hunk 1]
```diff
@@ -38,68 +38,6 @@
       "reference": "reference",
       "telephone": "00000000000"
     },
-    "allParties": [
```
Comment (lewisbirks) [inline e2e/fixtures/caseData/gatekeepingNoHearingDetails.json:41]: please revert as discussed

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsService.java [hunk 2]
```diff
@@ -47,6 +47,7 @@ public boolean hasEmptyDates(CaseData caseData) {
                 caseData.getCafcassDirections(),
                 caseData.getOtherPartiesDirections(),
                 caseData.getCourtDirections())
+            .filter(Objects::nonNull)
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsService.java:50]: As discussed I don't think this is the correct solution, the issue should be caused else where as it should not be possible to get here without having these directions, which would be populated in the notify gatekeeper event

Comment (lewisbirks) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsService.java:50]: As discussed we have worked out that the issue is some bad test data. Please update that test data and then everything should be fine.

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsServiceTest.java [hunk 3]
```diff
@@ -91,6 +91,20 @@ void shouldReturnFalseWhenThereAreNoEmptyDates() {
         assertThat(service.hasEmptyDates(caseData)).isFalse();
     }
 
+    @Test
+    void shouldReturnFalseWhenCafcassDirectionsAreNull() {
```
Comment (lewisbirks) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsServiceTest.java:95]: Can be reverted

**Review by lewisbirks — CHANGES_REQUESTED** [review]
FYI @samuelzerozero if you are interested

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsService.java (hunk 2 above)
Comment (toireasam) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsService.java]: changed based on discussion

**Review by lewisbirks — APPROVED** [review]

**Pull request merged**